js/ui/date_box/ui.date_box.mask.js

```diff
@@ -96,7 +96,10 @@ const DateBoxMask = DateBoxBase.inherit({
     },
 
     _shouldUseOriginalHandler(e) {
-        return !this._useMaskBehavior() || this.option("opened") || (e && e.altKey);
+        const keysToHandleByMask = ["backspace", "del"];
+        const isNotDeletingInCalendar = this.option("opened") && e && !keysToHandleByMask.includes(normalizeKeyName(e));
```

Comment:
IE11 doesn't support Array.prototype.includes

Reply:
Got it, thanks